### ISSUE TITLE
[TBC - Black Temple]Reliquary of Souls - Essence of Suffering (phase 1)

### ISSUE THREAD

**Issue by alexuktc** [dalaranwow/dalaran-wow#4503]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
3 major issues arise in this phase which make the fight challenging even for a group of geared level 80s. 

1. The boss' aggro mechanics are not only not correct, they are worse than they should be. As it stands it  randomly and on random intervals simply goes for the ranged dps (even though they are specifically instructed to outrange is as much as possible) or changes targets even though it shouldn't. Coupled with the other issues this is a very big problem.
2. The boss' damage is not working properly at all. The boss should be doing 1250 damage per second to whichever target it has (the only benefit tanks have over normal dps/healers here is the higher HP pool) and 2500 damage every 0.5 seconds during Enrage. Granted I am a rogue wearing leather, I got hit for 6886 normal damage (or a number close to it, it happened yday and I will not be able to test it until next reset).
3. The boss can crit and crush - during Frenzy I got crushed for 18900-19000 damage, killing me instantly. Our paladin tank had also died, along with several range dps who got chased for no reason.

**Expected behaviour**: Tell us what should happen instead
1. The boss should lock aggro onto the **nearest** target every 5 seconds. This was especially important as you could choose who would tank and when. Enrages were tanked by high agility (dodge) druids or rogues using evasion. Ranged dps was pretty much afk dps-ing on this phase with absolutely no fear of getting chased.
2. Pretty much explained in the upper section. Coupled with the aura that removes armor, defense and healing done the dps requirements become higher than a lot of WotLK raids.
3. The boss should be unable to crit and crush, as its aura leaves everyone very vulnerable to crits and crushes despite defense gear and talents.

**Steps to reproduce the problem**:

1. Go to BT
2. Start the boss
3. Observe the mentioned issues.

**Include proofs for this behaviour**
Sadly I am saved, so I cannot make a video or provide logs at this moment. It is very easy to reproduce, as it WILL happen on every attempt to kill this boss.

http://wowwiki.wikia.com/wiki/Reliquary_of_Souls - Official wowwiki on this boss. The boss in question is Essence of Suffering and all skills and points mentioned here are explained.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
npc=23418

While I know this might be low priority, lowering the damage and removing the random chasing of ranged dps should be enough to make the boss manageable without 50k+ dps in raid.

**Comment by Rushor** [comment]
Fixed with https://github.com/TrinityCore/TrinityCore/pull/18403 and https://github.com/TrinityCore/TrinityCore/pull/18470  Those fixed needs to be merged on the Live Realm. Thanks for the report.

**Comment by alexuktc** [comment]
Hello again. 

As can be found here https://github.com/TrinityCore/TrinityCore/pull/18403 This is supposedly already merged with the public realm. 

I did BT last night and RoS was the same as it was before. I have explained the issues above so no need to repeat myself. I was also instructed to make a new issue if the problem persists, but I think reopening this one should be enough.
